client_cli/src/main.rs

```diff
@@ -241,29 +220,8 @@ fn submit(
     let iroha_client = context.client_from_config();
     let instructions = instructions.into();
     let tx = iroha_client.build_transaction(instructions, metadata);
-    let transactions = if context.skip_mst_check() {
-        vec![tx]
-    } else {
-        match iroha_client.get_original_matching_transactions(
-            &tx,
-            RETRY_COUNT_MST,
-            RETRY_IN_MST,
-        ) {
-            Ok(original_transactions) if !original_transactions.is_empty() && Confirm::new()
-                .with_prompt("There are similar transactions from your account waiting for more signatures. \
-                            This could be because they weren't signed with the right key, \
-                            or because they're a multi-signature transactions (MST). \
-                            Do you want to sign these transactions (yes) \
-                            instead of submitting a new transaction (no)?")
-                .interact()
-                .wrap_err("Failed to show interactive prompt.")? => {
-                    original_transactions.into_iter().map(|transaction| {
-                        iroha_client.sign_transaction(transaction)
-                    }).collect()
-                }
-            _ => vec![tx],
-        }
-    };
+    let transactions = vec![tx];
```

Comment:
Initially we decided to defend ourselves from situation where there are multiple transactions returned by 'pending_transactions' endpoint. Currently there is only 1 transaction, so we shouldn't wrap it into Vec.